docker-compose.override.yml

```diff
@@ -4,13 +4,7 @@ services:
     command: python src/manage.py runserver 0.0.0.0:8000
     volumes:
       - .:/code
-      - /code/src/sous-chef
-      - /code/src/sous-chef/static
     expose:
       - "8000"
     ports:
```

Comment:
> I don't really understand how the Nginx container can redirect the request to the Web container through the port "8000" if the port is not exposed

This is because all the containers are on the same network and can see each other, so nginx can see port 8000 without any "expose" clause. The "expose" clause is to expose ports to the outside world (which you do correctly now).